### ISSUE TITLE
Why does PolynomialDecaySchedule default to power=1.0?

### ISSUE THREAD

**Issue by stas00** [pytorch/fairseq#2466]
## Question

This is not a bug report but a hparam choice question for `PolynomialDecaySchedule`.

We are adding a polynomial decay scheduler in transformers (https://github.com/huggingface/transformers/pull/6361) and I was told to match the defaults to `PolynomialDecaySchedule`.

Could you please explain why the default `power` is `1.0` and not another value that actually activates the polynomial decay?

Surely one can override it, but your recommendation is not to override it as it can be seen here:
https://github.com/pytorch/fairseq/blob/master/examples/mbart/README.md#finetune-on-en-ro
`--power` is left to the default value of `1.0`.

Thank you for your time.

**Comment by myleott** [comment]
This is based on the original BERT code, which implemented a linear decay via a polynomial schedule with power=1.0: https://github.com/google-research/bert/blob/f39e881b169b9d53bea03d2d341b31707a6c052b/optimization.py#L37

**Comment by stas00** [comment]
Thank you very much for your response and the link, @myleott!
